SchoolHousing/SchoolHousing/Models/AffordableHouse.cs

```diff
@@ -10,15 +10,15 @@ namespace AffordableHouseResponse
 {
     using System;
     using System.Collections.Generic;
-
+   
     using System.Globalization;
     using Newtonsoft.Json;
     using Newtonsoft.Json.Converters;
 
     public partial class AffordableHouse
     {
         [JsonProperty("community_area", NullValueHandling = NullValueHandling.Ignore)]
-        public string CommunityArea { get; set; }
+                public string CommunityArea { get; set; }
```

Comment:
This indentation is not correct... should leave it as it was.  Perhaps this was a typo.